### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by fernandog**
Fixes some issues I found with code inspection

- ~~Missing log type~~
- ~~self.imdbid doesn't exist~~
- ~~Last line too long in series.py~~
- Duplicate keys
- Missing reference
- Missing parentheses

### CODE REVIEW

File: medusa/databases/main_db.py [hunk 1]
```diff
@@ -235,7 +235,6 @@ def fix_indexer_show_statues(self):
             'planned': 'Continuing',
             'in production': 'Continuing',
             'pilot': 'Continuing',
-            'returning series': 'Continuing',
```
Comment (ratoaq2) [inline medusa/databases/main_db.py]: Why was this removed?

File: medusa/metadata/generic.py [hunk 2]
```diff
@@ -953,7 +953,7 @@ def size_str_to_int(x):
 
         try:
             search = tmdb.Search()
-            for show_name in show.get_all_possible_names:
+            for show_name in show.get_all_possible_names():
```
Comment (ratoaq2) [inline medusa/metadata/generic.py]: This is wrong. A property with a method name?

File: medusa/tv/series.py [hunk 3]
```diff
@@ -1715,7 +1715,8 @@ def to_json(self, detailed=True):
                 ('seasons', OrderedDict([]))
             ]))
             episodes = self.get_all_episodes()
-            result['seasons'] = [list(v) for _, v in groupby([ep.to_json() for ep in episodes], lambda item: item['season'])]
+            result['seasons'] = [list(v) for _, v in groupby([ep.to_json() for ep in episodes],
```
Comment (ratoaq2) [inline medusa/tv/series.py]: This changes will conflict with my api v2 PR

File: medusa/databases/main_db.py (hunk 1 above)
Comment (fernandog) [inline medusa/databases/main_db.py]: Duplicate keys

File: medusa/tv/series.py (hunk 3 above)
Comment (fernandog) [inline medusa/tv/series.py]: i will revert this then

File: medusa/tv/series.py [hunk 4]
```diff
@@ -1048,8 +1048,8 @@ def make_ep_from_file(self, filepath):
                     if not cur_ep:
                         raise EpisodeNotFoundException
                 except EpisodeNotFoundException:
-                    logger.log(u'{indexerid}: Unable to figure out what this file is, skipping {filepath}',
-                               indexerid=self.indexerid, filepath=filepath)
+                    logger.warning(u'{indexerid}: Unable to figure out what this file is. Skipping {filepath}',
```
Comment (labrys) [inline medusa/tv/series.py]: This was originally an error, so should probably continue to be error level.
